lib/rules/disallow-space-after-keywords.js

```diff
@@ -63,9 +63,13 @@ module.exports.prototype = {
 
     check: function(file, errors) {
         file.iterateTokensByTypeAndValue('Keyword', this._keywords, function(token) {
+            var nextToken = file.getNextToken(token);
+            if (token.value === 'else' && nextToken.value === 'if') {
```

Comment:
Perhaps use something like what is found here: fef808ea42809fb7ea8d9244913bfcb478c55313